apps/exactwords/app.js

```diff
@@ -0,0 +1,226 @@
+  minutesRough = MINUTES_ROUGH_ARRAY[Math.round((m + 0 ) / 5)];
+  minutesAccurate = MINUTES_ACCURATE_ARRAY[m % 5]; 
+  
+  words = minutesAccurate + minutesRough + hourWord + partDayWord;
+  words = words.charAt(0).toUpperCase() + words.slice(1);
+  return words;
+}
+
+function wordsFromDayMonth(day, date, month)
+{
+  // Tests
+
+// Example 12:00 = Twelve
+//  New Year's Day
+//  date = 1;
+//  month = 0;
+//  on the Ides of March
+//  date = 15;
+//  month = 2;
+// , ERROR C Nonsense in BASIC
+//  date = 1;
+//  month = 3;
+//  - O'Canada
+//  date = 1;
+//  month = 6;
+//  -  on Halloween
+//  date = 31;
+//  month = 9;
+//  -  Christmas Eve
+//  date = 24;
+//  month = 11;
+//  -  Christmas Day
+//  date = 25;
+//  month = 11;
+//  -  Boxing day
+//  date = 26;
+//  month = 11;
+//  New Year's eve
+//  date = 31;
+//  month = 11;
+//  longest
+//  date = 29;
+//  month = 10;
+  
+  
+  const DAY_WORD_ARRAY = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  const DATE_WORD_ARRAY = ["zero", "first", "second", "third", "fourth", "fifth", "sixth",  "seventh","eighth", "ninth", "tenth", "eleventh", "twelfth", "thirteenth",   "fourteenth", "fifteenth","sixteenth",     "seventeenth",  "eighteenth",   "nineteenth",     "twentieth",     "twenty-first", "twenty-second", "twenty-third","twenty-fourth", "twenty-fifth", "twenty-sixth", "twenty-seventh", "twenty-eighth", "twenty-ninth", "thirtieth",     "thirty-first"];
+  const MONTH_WORD_ARRAY = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+  var words = " ";
+  words = DAY_WORD_ARRAY[day] + ", " + DATE_WORD_ARRAY[date] + " of " + MONTH_WORD_ARRAY[month];
+  if ((date == 1) && (month == 0)) {
+    words = "New Year's Day";
+  } else if ((date == 15) && (month == 2)) {
+    words = DAY_WORD_ARRAY[day] + " on the Ides of March";
+  }  else if ((date == 1) && (month == 3)) {
+    words = DAY_WORD_ARRAY[day] + ", ERROR C Nonsense in BASIC";
+  } else if ((date == 1) && (month == 6)) {
+    words = DAY_WORD_ARRAY[day] + "  - O'Canada";
+  } else if ((date == 31) && (month == 9)) {
+    words = DAY_WORD_ARRAY[day] + " - on Halloween";
+  } else if ((date == 24) && (month == 11)) {
+    words = "Christmas Eve";
+  } else if ((date == 25) && (month == 11)) {
+    words = "Christmas Day";
+  } else if ((date == 26) && (month == 11)) {
+    words = "Boxing Day";
+  } else if ((date == 31) && (month == 11)) {
+    words = "New Year's eve";
+  } 
+  return words;
+}
+
+function draw() {
+  var x = g.getWidth()/2;
+  var y = g.getHeight()/2;
+  g.reset();
+  
+  var d = new Date();
+  var h = d.getHours();
+  var m = d.getMinutes();
+  var day = d.getDay();
+  var date = d.getDate();
+  var month = d.getMonth();
+
+  var timeStr = wordsFromTime(h,m);
+  var dateStr = wordsFromDayMonth(day, date, month);
+  
+  // draw time
+  // TO DO: Set Theme
+  g.setBgColor("#000000");
+  g.setColor("#ffffff");
```

Comment:
I think you could just do 

```suggestion
  g.setBgColor(g.theme.bg);
  g.setColor(g.theme.fg);
```

Here to make it compatible with dark/light (and other) system themes.